### ISSUE TITLE
Blog article on repository find methods

### ISSUE THREAD

**Issue by grahamwallis** [odpi/egeria#1621]

**Comment by grahamwallis** [comment]
Please see draft post https://odpilf.lfprojects.linuxfoundation.org/wp-admin/post.php?post=91434

**Comment by mandy-chessell** [comment]
I can not see it - I can only view the ones that are published

**Comment by grahamwallis** [comment]
Hmm.. that's odd. I think I've done everything to make it 'public' in review state.

<img width="358" alt="image" src="https://user-images.githubusercontent.com/17125304/65580793-3f227780-df72-11e9-862b-1e4d9845bae9.png">

I'm not sure what other controls are available. 
@cmgrote @jmertic - Any ideas?

**Comment by cmgrote** [comment]
I don't know if it's just a WordPress peculiarity or what -- I can see our entry listed under "Pending", but I have no way of opening it or previewing it.  Instead, if I hover over the entry I get options for "Clone" or "New Draft".  It seems my only way to preview is to "Clone" the post, which then creates a copy under my own name (as draft), which I can then read and review.  Once I'm done I can just delete this draft under my own name, and your original remains unaffected.

<img width="892" alt="Screenshot 2019-09-25 at 09 07 27" src="https://user-images.githubusercontent.com/5864283/65581806-0edbd880-df74-11e9-81ae-724c87ca59f1.png">

(Very convoluted, but it more or less works for what we're after...)

**Comment by planetf1** [comment]
I can't see it either, but nor can I see @moyabrannan 's posts, nor can anyone see mine.
The only current option is to share a pdf/export but I agree it would be really useful if we could get acl/workflow sorted to allow this

**Comment by cmgrote** [comment]
My quick review comments:

- My personal preference is to `mono-style` code references, eg. `findEntitiesByProperty()` rather than just findEntitiesByProperty().  You can easily apply this inline in the visual editor the same way you would in Markdown -- just put single back-ticks around the word / code snippet you want to `mono-style`.
- In the first section, you have an "an" that should be an "a": "The first method accepts a~~n~~ 'match properties' object which can be used to specify a 'match' value for each property."
- Your second section is a bit misleading to me...  The repository connector interface (the MetadataCollection interface, as you introduce it) _always_ expects a regex for string properties: the `getExactMatchRegex()` helper is just giving a regex that ensures it matches exactly the string provided.  The way it's written at the moment it sounds as if sometimes it expects a regex and other times it does not...
- Last section, missing an "of" and I suspect we should really say that _all_ need to be unescaped, not _any_ (which implies a bit of randomness): "However, if any match properties were escaped prior to the search, the OMAS would need to 'unescape' **all of** them."

**Comment by planetf1** [comment]
Nice review workaround (clone) @cmgrote!

**Comment by grahamwallis** [comment]
Thanks Chris

- I have mono-styled all references to class and method names.
- Fixed the 'an match properties' typo.
- I think I've clarified the misleading part.
- I think I've disambiguated the last part by saying 'However, if any match properties were escaped prior to the search, the OMAS would need to 'unescape' those match properties.'
- I also made a few other changes to improve the flow - I had rephrased parts of the original to try to improve the 'readability' score but I think the result was a bit tortuous - hopefully it reads a little better now.

Re-submitted for review.
